WIP-CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+### Changed
```

Comment:
Mention the migration script in "release notes" or is "run migration scripts" always implied in the upgrade process?

Reply:
It's part of the migration document https://github.com/elastisys/compliantkubernetes-apps/blob/f1bd76dcd40ce4940043e37dc9b132ae2ea5e691/migration/v0.15.x-v0.16.x/upgrade-apps.md which is where we have all these instructions :slightly_smiling_face:

Reply:
Good question, we seem to sometimes have it at least https://github.com/elastisys/compliantkubernetes-apps/blob/main/CHANGELOG.md#release-notes
But we should definitely always have that in the release notes. Good catch